projects/openjdk.org/package.yml

```diff
@@ -64,12 +68,19 @@ build:
     - run: sed -i -e's/c..filt/llvm-cxxfilt/' toolchain.m4
       working-directory: make/autoconf
       if: linux
-    - bash configure $ARGS
-      --with-boot-jdk="$BOOT_JDK_DIR"
-      --with-vendor-version-string="$(pkgx --version | grep -o '[0-9]\+\.[0-9]\+\.[0-9]\+')"
-      --with-extra-cflags="$CFLAGS"
-      --with-extra-cxxflags="$CXXFLAGS"
-      --with-extra-ldflags="$LDFLAGS"
+    - |
+      if [ {{version.major}} -gt 11 ]; then
+        version_args=--with-version-build={{version.minor}}
+      else
+        version_args="--disable-hotspot-gtest --with-conf-name=release --with-jvm-features=shenandoahgc"
+      fi
```

Comment:
@jhheider this was the best solution I thought of to condition the args. LMK if you have a better suggestion

Reply:
probably use two step lines:
```yaml
  - run: version_args=--with-version-build={{version.minor}}
    if: '>=12'
  - run: version_args="--disable-hotspot-gtest --with-conf-name=release --with-jvm-features=shenandoahgc"
    if: <12
```

Reply:
oh wow. I guess this means pkgx compiles a single shell script with all the commands before running for this to work then. Will try, thanks.

Reply:
yep, script-arrays were added last fall, and allow for better control of steps.